From 3e24d34d9c36b61973871b0dadb6c11e798348f4 Mon Sep 17 00:00:00 2001
From: Ben Cooksley <bcooksley@kde.org>
Date: Mon, 7 Feb 2022 06:28:21 +1300
Subject: [PATCH] Purge all references to download.kde.org, replacing them with
 the correct endpoint of autoconfig.kde.org.

CCMAIL: plasma-devel@kde.org
CCMAIL: distributions@kde.org
(cherry picked from commit 2c39c2051d9c7fc9e2e5c28c1d0ed44ea582ccd6)
---
 applets/comic/comic.knsrc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/applets/comic/comic.knsrc b/applets/comic/comic.knsrc
index 264381972..bfa9a5dda 100644
--- a/applets/comic/comic.knsrc
+++ b/applets/comic/comic.knsrc
@@ -47,7 +47,7 @@ Name[x-test]=xxComicsxx
 Name[zh_CN]=漫画
 Name[zh_TW]=漫畫
 
-ProvidersUrl=https://download.kde.org/ocs/providers.xml
+ProvidersUrl=https://autoconfig.kde.org/ocs/providers.xml
 Categories=Plasma Comic
 TargetDir=plasma/comics
 Uncompress=kpackage
-- 
GitLab

